browser/src/control/Control.DocumentRepair.js

```diff
@@ -58,7 +58,7 @@ L.Control.DocumentRepair = L.Control.extend({
 										{
 											id: 'cancel',
 											type: 'pushbutton',
-											text: '~Cancel',
+											text: _('~Cancel'),
```

Comment:
Please remove the ~, it is removed programmatically anyway, but then we'd get translations from Cancel for free.

Reply:
And a few lines below, make OK tanslatable, e.g. in Spanish it's Acceptar.
_('OK')
And actually the dialog title is untranslatable. too: 'Repair Document' :)